src/aliceVision/sfm/LocalBundleAdjustmentGraph.cpp

```diff
@@ -657,24 +657,26 @@ std::size_t LocalBundleAdjustmentGraph::addIntrinsicEdgesToTheGraph(const sfmDat
 
     for(const auto& x : _nodePerViewId) // for each reconstructed view in the graph
```

Comment:
`for(const auto& x : _nodePerViewId) // for each reconstructed view in the graph. Warning: "_nodePerViewId" already contains the "newReconstructedViews"`